Apps/gitea-mirror/README.md

```
@@ -40,12 +45,15 @@ All of these options can be configured through the UI after installation:
 
 While most configuration can be done through the UI, you can also pre-configure the application using environment variables:
 
-- `JWT_SECRET`: Secret key for JWT authentication (important for security)
+- `BETTER_AUTH_SECRET`: Secret key for Better Auth authentication (important for security)
+- `BETTER_AUTH_URL`: Authentication URL (defaults to http://0.0.0.0:4321)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)